ffi/capi/src/exemplar_chars.rs

```diff
@@ -51,7 +51,21 @@ pub mod ffi {
             FnInStruct
         )]
         #[diplomat::attr(supports = fallible_constructors, named_constructor = "main")]
-        pub fn try_new_main(
+        #[cfg(feature = "compiled_data")]
+        pub fn try_new_main(locale: &Locale) -> Result<Box<ExemplarCharacters>, DataError> {
```

Comment:
"using compiled data"

Reply:
I'm not adding docs in this PR to types that have no docs on them, we should flag that for a separate issue.